main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  Fish
+//  action 1: the fish swims
+fish.swim();
+//  action 2: the fish blows bubbles
+fish.blowBubble();
+//  action 3: the fish eats food
+fish.eatFood();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  Cat
+//  action 1: the cat meows
+cat.meow();
+//  action 2: the cat chases the mouse
+cat.chaseMouse();
+//  action 3: the cat drinks milk
+cat.drinkMilk();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  Baby
+//  action 1: the baby cries
+baby.cry();
+//  action 2: the baby falls asleep gently
+baby.fallAsleepGently();
+//  action 3: the baby eats baby food
+baby.eatBabyFood();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  Musician
+//  action 1: the musician tunes their instrument
+musician.tuneInstrument();
+//  action 2: the musician practices to a metronome
+musician.practiceToMetronome();
+//  action 3: the musician writes a song
+musician.writeSong();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: Painter
+//  action 1: the painter picks the colors
+painter.pickColor();
+//  action 2: the painter draws on the canvas
+painter.drawOnCanvas():
+//  action 3: the painter cleans their paintbrushes
+painter.cleanPaintbrush();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  Horse
+//  action 1: the horse neighs
+horse.Neigh();
+//  action 2: the horse eats food
+horse.eatFood();
+//  action 3: the horse gallops
+horse.Gallop();

```

Comment:
In this course, function names begin with a lower case letter, not an upper case letter.
upper case letters are reserved for Type names, which have not been covered yet
- [ ] revise